vmodules/commands/about.ts

```diff
@@ -15,12 +14,12 @@ const metadata = {
 };
 
 metadata.run = async (m, args, gcfg) => {
-  const bot = memory.client;
+  const bot = memory.client!;
```

Comment:
the bot will be instantiated at this point, right? we dont need to worry about a null Vector object?